Society for Neuroscience Abstract 2016.js

```diff
@@ -0,0 +1,446 @@
+{
+	"translatorID": "1ed36c21-de68-4ba4-a27f-d724ce258db4",
+	"label": "Society for Neuroscience Abstract 2016",
+	"creator": "Kouchi C. Nakamura, PhD; kouichi.c.nakamura@gmail.com",
+	"target": "^https?://www\\.abstractsonline\\.com/pp8/#!/(4036|4376)/presentation/",
```

Comment:
Can we extend this target more? I see there are other conferences on this platform, e.g. http://www.abstractsonline.com/pp8/#!/4292/presentation/6761 , http://www.abstractsonline.com/pp8/index.html#!/3658/presentation/37598 and possibly they all have their data structured similar.

Reply:
Thank you for finding other meeting pages.

This translator might work with the former with only a few modifications (or possibly no modification). But it is not compatible with the latter (creators will be empty). As I guessed, the formatting is dependent to meetings to some extent. It's beyond my capability to adapt many others.

Reply:
As I gussed the translator worked for http://www.abstractsonline.com/pp8/#!/4292/presentation/6761 with only a minor issue that first author's first name starts with `&nbsp`;



```json
               {
                 "firstName": "&nbsp;S",
                 "lastName": "Tian",
                 "creatorType": "author"
               }
```

The translator simply failed 
 for http://www.abstractsonline.com/pp8/index.html#!/3658/presentation/37598.

Reply:
Let us try to cover all these cases. At least we shouldn't exclude them here in the target. I think even we can improve author parsing to work in most of the cases...